src/Exceptions/HttpVariantAlsoNegotiates.php

```diff
@@ -8,4 +8,3 @@ class HttpVariantAlsoNegotiates extends HttpException {
     public const DESCRIPTION = "Indicates that the server has an internal configuration error: the chosen variant resource is configured to engage in transparent content negotiation itself, and is therefore not a proper end point in the negotiation process.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/506";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces